src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -87,7 +89,8 @@ def copy_dir(src: Path, tgt: Path, glob_pattern: str, should_touch_init: bool =
 FILTER_CONFIG = qa_config.FILTER_CONFIG if getattr(qa_config, 'FILTER_CONFIG', False) else None
 
 
-RUN_ID = MODEL_CONFIG.RUN_ID
+RUN_ID = MODEL_CONFIG.RUN_ID if getattr(MODEL_CONFIG, 'RUN_ID', False) else None
+RUN_IDs = MODEL_CONFIG.RUN_IDs if getattr(MODEL_CONFIG, 'RUN_IDs', False) else None
```

Comment:
Maybe "xor" could help to make sure exactly one is set

```suggestion
RUN_ID = MODEL_CONFIG.RUN_ID if getattr(MODEL_CONFIG, 'RUN_ID', False) else None
RUN_IDs = MODEL_CONFIG.RUN_IDs if getattr(MODEL_CONFIG, 'RUN_IDs', False) else None
bool(RUN_ID) != bool(RUN_IDS)
```